### PR TITLE
Kibana Dashboard

### PR DESCRIPTION
**Pull request by tybalex**
This PR contains a few major changes:
1. documentation and template of Kibana dashboard (and Grafana), and how to set up alerts in Kibana
2. changes that allow user to launch training jobs via the Dev Tool in Kibana.

minor changes include:
1. adapt nulog-inference-service to opendistro-es endpoint
2. adapt training-controller to opendistro-es endpoint
3. decorate a few functions. add some comments and TODOs.

This PR should be reviewed and merged after PR #4

### CODE REVIEW

**Review by AmartC — DISMISSED** [review]
Looks good for most part but please address the environmental variables to be consistent with what nulog-train job is using.

File: src/dashboard/Grafana/README.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
+## deploy grafana with helm
```
Comment (sanjay920) [inline src/dashboard/Grafana/README.md:1]: we should remove this altogether along with sock-shop-live-dashboard.json

**Review by sanjay920 — APPROVED** [review]

**Review by AmartC — APPROVED** [review]
LGTM. Just make sure to update your branch with v0.0 before pushing.

File: src/dashboard/Grafana/README.md (hunk 1 above)
Comment (tybalex) [reply src/dashboard/Grafana/README.md]: removed the Grafana folder.

**Pull request merged**